github-users-web-client/src/main/java/com/xemantic/githubusers/web/navigation/UserSelectionNavigator.java

```diff
@@ -37,22 +37,21 @@
 @Singleton
 public class UserSelectionNavigator {
 
-  private final EventBus eventBus;
+  private final Observable<UserSelectedEvent> eventBus;
```

Comment:
But `userSelected$` might be a good choice too! do whatever sounds better to you!